microsoft-edge/devtools-guide-chromium/console/index.md

```diff
@@ -46,7 +46,7 @@ The fastest way to directly open the **Console** is to select `Control`+`Shift`+
 
 ## Look up error messages on the web from the Console
```

Comment:
It's very important we lock on the section titles since we essentially hard-code those strings in to the Welcome tool in the DevTools (using go links but still, I like to avoid changing them if possible) so I'd suggest we go for a shorter, more concise title like:

```suggestion
## Search errors on the web from the Console
```

Reply:
I'm good with this title, though I'd like to keep "error messages" as the term we use within the content.